src/test/java/jenkins/plugins/office365connector/HttpWorkerTest.java

```diff
@@ -44,99 +43,82 @@
-		connection.close();
-	}
-	
-	@Test
-	public void testSendingMultipleWebhooks() throws IOException, InterruptedException {
-		ExecutorService executorService = Executors.newCachedThreadPool();
-		HttpWorker worker1 = new HttpWorker("http://localhost:8000/test1", "test1body", 30000, 1, Mockito.mock(PrintStream.class));
-		HttpWorker worker2 = new HttpWorker("http://localhost:8000/test2", "test2body", 30000, 1, Mockito.mock(PrintStream.class));
-		executorService.submit(worker1);
-		executorService.submit(worker2);
-		executorService.shutdown();
-		executorService.awaitTermination(5, TimeUnit.SECONDS);
-		Assert.assertTrue(MyHandler.getTest1Result());
-		Assert.assertTrue(MyHandler.getTest2Result());
-	}
-	
-	@Test
-	public void testMutipleTriesWorker() throws InterruptedException {
-		ExecutorService executorService = Executors.newCachedThreadPool();
-		HttpWorker worker = new HttpWorker("http://localhost:8000/retry-test", "test1body", 30000, retries, Mockito.mock(PrintStream.class));
-		executorService.submit(worker);
-		executorService.shutdown();
-		executorService.awaitTermination(5, TimeUnit.SECONDS);
-		Assert.assertTrue(MyHandler.getRetryTestResult());
-	}
-	
-	static class MyHandler implements Container {
-		
-		static int trialTestRetries = 0;
-		
-		static boolean test1Result = false;
-		
-		static boolean test2Result = false;
-		
-		public static boolean getRetryTestResult() {
-			return trialTestRetries == retries;
-		}
-		
-		public static boolean getTest1Result() {
-			return test1Result;
-		}
-		
-		public static boolean getTest2Result() {
-			return test2Result;
-		}
-		
+    WebhookJobProperty property;
+    private static Connection connection;
+
+    @BeforeClass
+    public static void setup() throws IOException {
+        Container container = new MyHandler();
+        Server server = new ContainerServer(container);
+        connection = new SocketConnection(server);
+        SocketAddress address = new InetSocketAddress(8000);
+        connection.connect(address);
+    }
+
+    @AfterClass
+    public static void destroy() throws IOException {
+        connection.close();
+    }
+
+    @Test
+    public void testSendingMultipleWebhooks() throws IOException, InterruptedException {
+        ExecutorService executorService = Executors.newCachedThreadPool();
+        HttpWorker worker1 = new HttpWorker("http://localhost:8000/test1", "test1body", 30000, Mockito.mock(PrintStream.class));
+        HttpWorker worker2 = new HttpWorker("http://localhost:8000/test2", "test2body", 30000, Mockito.mock(PrintStream.class));
+        executorService.submit(worker1);
+        executorService.submit(worker2);
+        executorService.shutdown();
+        executorService.awaitTermination(5, TimeUnit.SECONDS);
+        Assert.assertTrue(MyHandler.getTest1Result());
+        Assert.assertTrue(MyHandler.getTest2Result());
+    }
+
+    static class MyHandler implements Container {
+
+        static int trialTestRetries = 0;
+
```

Comment:
removes ``testMutipleTriesWorker`` as this test in fact validated only getter method what now does not make sense